src/plugins/discover/public/application/components/sidebar/discover_index_pattern.test.tsx

```diff
@@ -58,6 +58,7 @@ function getIndexPatternPickerOptions(instance: ShallowWrapper) {
 function selectIndexPatternPickerOption(instance: ShallowWrapper, selectedLabel: string) {
   const options: Array<{ label: string; checked?: 'on' | 'off' }> = getIndexPatternPickerOptions(
     instance
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
An easy win in terms of omitting `any` and improved "docs by types" could be improving `getIndexPatternPickerOptions` (L54)  in the following way:
```TypeScript
function getIndexPatternPickerOptions(
  instance: ShallowWrapper
): Array<{ label: string; key: string; value: string; checked: string }> {
  return getIndexPatternPickerList(instance).prop('options');
}
```

Reply:
I'd like to push that into another PR nevertheless. I think we have still a lot of follow up PRs to actually "clean-up" typings. I mainly just wanted to enable the linting rules here and ignore those `any` for now. I'd expect us to clean up all of them (via Angular removal or otherwise) in follow up PRs anyway :) (I'll store your comment though for the further cleanup PRs I am opening).

Reply:
got it